### PR TITLE
Include send btc transaction function

### PR DESCRIPTION
**Pull request by Imamah-Zafar**

### CODE REVIEW

**Review by abdulhaseeb4239 — CHANGES_REQUESTED** [review]

File: src/transactions/sendBtcTransaction.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { createUnsecuredToken, Json } from 'jsontokens';
+import { BitcoinNetwork } from '../provider';
+
+export interface SendBtcTransactionPayload {
+  network: BitcoinNetwork;
+  satsAmount: string;
```
Comment (yknl) [inline src/transactions/sendBtcTransaction.ts:6]: amountSats makes more sense here

File: src/transactions/sendBtcTransaction.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  recipientAddress: string
+  message: string;
```
[the remark this answers is not in the document]
Comment (Imamah-Zafar) [reply src/transactions/sendBtcTransaction.ts:8]: all of the previous implemented functions do have `message` attribute in its payload. For reference check GetAddressPayload, SignTransactionPayload, SignMessagePayload

for the sake of being consistent, shouldnt the message attribute also be for `SendBtcTransactionPayload`

[the remark this answers is not in the document]
Comment (abdulhaseeb4239) [reply src/transactions/sendBtcTransaction.ts:8]: I am not sure why we need a message for a BTC transaction request, we should make it optional if anything

**Review by abdulhaseeb4239 — APPROVED** [review]

**Comment by DuskaT021** [comment]
This is tested together with Imamah and it can be merged

**Pull request merged**